<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-I6EP1Q3" failures="2" tests="4" name="Test" time="39.114" errors="0" timestamp="2023-12-05T19:00:36 EST">
  <testcase classname="api.test.UserTest2" name="testPostUser" time="37.217">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTest2.testPostUser(UserTest2.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testPostUser -->
  <testcase classname="api.test.UserTest2" name="testGetUser" time="0.257">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTest2.testGetUser(UserTest2.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testGetUser -->
  <testcase classname="api.test.UserTest2" name="testUpdtaeUser" time="0.427"/>
  <testcase classname="api.test.UserTest2" name="testDeleteUser" time="0.22"/>
</testsuite> <!-- Test -->
